src/mds/Server.cc

```diff
@@ -6882,6 +6882,13 @@ void Server::handle_client_getvxattr(const MDRequestRef& mdr)
       // since we only handle ceph vxattrs here
       r = -CEPHFS_ENODATA; // no such attribute
     }
+  } if (xattr_name == "ceph.dir.subvolume"sv) {
```

Comment:
Does fetching the xattr via `CEPH_MDS_OP_GETATTR` still work (i.e., if the MDS does not have `CEPHFS_FEATURE_OP_GETVXATTR`)?

Reply:
yes, that part still works
the `getvxattr` is just a refactor of the main [`getxattr` on the client side](https://github.com/ceph/ceph/blob/9a76414d57f844b5b50dc9ca583c6f1e6a420fa1/src/client/Client.cc#L13745)

Reply:
OK. Just curious if this is covered in an existing test.

Reply:
the **get** part isn't addressed in existing tests

Reply:
> the **get** part isn't addressed in existing tests

When we backport this, then are the relevant tests covered?